File: app/src/main/java/io/github/ovso/righttoknow/ui/main/news/NewsAdapterDataModel.java
package io.github.ovso.righttoknow.ui.main.news;

import io.github.ovso.righttoknow.framework.adapter.BaseAdapterDataModel;
import io.github.ovso.righttoknow.ui.main.news.model.News;

/**
 * Created by jaeho on 2017. 9. 2
 */

public interface NewsAdapterDataModel extends BaseAdapterDataModel<News> {
  void setOnItemClickListener(OnNewsRecyclerItemClickListener<News> listener);
}

File: app/src/main/java/io/github/ovso/righttoknow/ui/main/news/NewsAdapterDataModel.kt
package io.github.ovso.righttoknow.ui.main.news

import io.github.ovso.righttoknow.framework.adapter.BaseAdapterDataModel
import io.github.ovso.righttoknow.ui.main.news.model.News

interface NewsAdapterDataModel : BaseAdapterDataModel<News> {
  fun setOnItemClickListener(listener: OnNewsRecyclerItemClickListener<News>)
}